### ISSUE TITLE
[Accessibility] - Not able to swipe content on (BankID)

### ISSUE THREAD

**Issue by joaooliveiramanteigas** [Adyen/adyen-web#1621]
Describe the bug:

User Story: 
As a customer I am not able to access the different payment alternatives with in the bankid verification using swipe and double click when screen reader is activated.

Description:
To meet the requirements of WCAG 2.1 [[2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#keyboard)] the customer needs to be able to use keyboard with swipe and double click to reach navigation elements.

Steps to reproduce the behaviour:

Pay with card related with bankid provider.
You should be redirected to the page shown in the picture.

Expected behaviour:

With an iPhone with VoiceOver on:
Settings > Accessibility > VoiceOver
When customer is trying to navigate these elements they must be reachable by swipe.

Image
Image displaying where the bug presides.
![image](https://user-images.githubusercontent.com/85570868/172637977-71b6d114-20f3-4107-b277-b1b5cd1a294f.png)

Smartphone (please complete the following information):

Device: iPhone 8
OS: iOS14.5
Browser: Chrome
Additional Context
We are using the Web Components/Drop-in v5.16.0

Thank you in advance.

**Comment by m1aw** [comment]
This issue should be raised with Vips/BankID, this is third-party to Adyen.
